src/tools/coverage-dump/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "coverage-dump"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+anyhow = "1.0.71"
+md5 = { package = "md-5" , version = "0.10.5" }
+miniz_oxide = "0.7.1"
+regex = "1.8.4"
+rustc-demangle = "0.1.23"
```

Comment:
All of these version numbers are just copied from whatever happened to be in `Cargo.lock` at the time, to avoid bumping the existing versions.